plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -324,6 +331,7 @@ func ClusterRoleBindings() []rbac.ClusterRoleBinding {
 		rbac.NewClusterBinding("cluster-admin").Groups(user.SystemPrivilegedGroup).BindingOrDie(),
 		rbac.NewClusterBinding("system:discovery").Groups(user.AllAuthenticated, user.AllUnauthenticated).BindingOrDie(),
 		rbac.NewClusterBinding("system:basic-user").Groups(user.AllAuthenticated, user.AllUnauthenticated).BindingOrDie(),
+		rbac.NewClusterBinding("system:cluster-facilities").Groups(user.AllAuthenticated, user.AllUnauthenticated).BindingOrDie(),
```

Comment:
Oh, right. If you're not authenticated, you're not ready to be ordering off the menu anyway.
It will likely take me until tomorrow to submit an updated patch.